### PR TITLE
Update action paths post login on campaigns

### LINKED ISSUE #1408: auth - failed log in on pitch page takes the user to a 404
To recreate:
- go to https://www.dosomething.org/campaigns/project-shutdown  as an anonymous user
- click sign up
- give an invalid email/password combo

I've only see it happen on pitch pages not when trying to log in on the homepage.

### PR DESCRIPTION
**Pull request by angaither**
Fixes #1408

### CODE REVIEW

File: lib/modules/dosomething/dosomething_user/dosomething_user.module [hunk 1]
```diff
@@ -235,6 +235,7 @@ function dosomething_user_login_submit($form, &$form_state) {
       dosomething_signup_user_signup($nid);
     }
   }
+  print_r($_SERVER['HTTP_REFERER']);
```
Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_user/dosomething_user.module]: :bomb:

**Comment by angaither** [comment]
AHH. :dancer: removed!

**Comment by aaronschachter** [comment]
Looks great.  Does the change made in #1415 need to be adjusted or is that good to go to?

**Comment by angaither** [comment]
No need removing it, that'll take care of logins on other content types.

**Comment by aaronschachter** [comment]
value added :+1:

**Pull request merged**